int32 bushId
bool isOnFire
int32 fireIntensity
